injected/src/features/broker-protection/utils.js

```diff
@@ -240,3 +240,19 @@ export async function hashObject(profile) {
 
     return hashHex;
 }
+
+/**
+ * @param {unknown} input
+ * @return {input is Object}
+ */
+export function isObject(input) {
```

Comment:
This feels like it should be in the general utils possibly? Kinda shocked we don't have this already.

Reply:
yep will move them both